test/runtests.jl

```diff
@@ -188,6 +188,16 @@ end
             end
         end
     end
+
+    srcdir = joinpath(@__DIR__, "BustedPackage", "src")
+    clean_folder(srcdir)
+    cmdstr = "using BustedPackage; BustedPackage.greet()"
+    cd("BustedPackage") do
+        run(`$(Base.julia_cmd()) --startup-file=no --code-coverage=user --project -e $cmdstr`)
+    end
+    bustedfile = joinpath(srcdir, "parseerr.jl")
+    @test_throws Base.Meta.ParseError("parsing error in $bustedfile, successfully handled up to line 6") process_file(bustedfile, srcdir)
```

Comment:
```suggestion
    @test_throws Base.Meta.ParseError("parsing error in $bustedfile:7: space before \"[\" not allowed in \"i [\"") process_file(bustedfile, srcdir)
```

Reply:
@timholy sorry my change proposal was not complete, the test needs to be adapted, too (the above might work, but I couldn't test it yet).

Thank you for your patience and willingness to help. If you prefer me to fix the test (you already went way beyond anything to be expected), let me know, I'll do it later today.